File: Upcoder/Khối 11/Chương 1 Làm Quen/Tinh Sin/TKN_K11_Sin.py
import math

a, x = map(float, input().split())
if x < -math.pi or x > math.pi:
    exit(1)

S = (a + math.sin(x)) / math.sqrt(a**2 + x**2 + 1)
S = round(S, 6)
print(S)

File: Upcoder/Khối 11/Chương 1 Làm Quen/Tinh Sin/TKN_K11_Sin.cs
using System;
namespace tinhSin
{
    public class Program
    {
        static void Main()
        {
            string input = Console.ReadLine();
            string[] inputs = input.Split(' ');
            float a = float.Parse(inputs[0]);
            float x = float.Parse(inputs[1]);
            if (x < -Math.PI || x > Math.PI)
            {
                return;  // Kết thúc chương trình với mã lỗi
            }
            // Tính giá trị biểu thức S
            float S = (a + (float)Math.Sin(x)) / (float)Math.Sqrt(a * a + x * x + 1);
            Console.Write("{0:0.000000}", S);
        }
    }
}